code/modules/holodeck/computer.dm

```diff
@@ -92,7 +92,7 @@ GLOBAL_LIST_INIT(typecache_holodeck_linked_floorcheck_ok, typecacheof(list(/turf
 
 /obj/machinery/computer/holodeck/LateInitialize()//from here linked is populated and the program list is generated. its also set to load the offline program
 	linked = GLOB.areas_by_type[mapped_start_area]
-	bottom_left = locate(linked.x, linked.y, src.z)
+	bottom_left = locate(linked?.x, linked?.y, src.z) // SKYRAT EDIT: Fixing CI with our maps
```

Comment:
What happens if that linked doesn't work? I feel like the Holodeck won't work, then, and that's not really good news.

Reply:
If linked didn't work before, it threw a runtime. Now it just silently fails. The issue with the runtime is that it will make it fail CI - it loads up runtimestation locally, which has no holodeck, but the testing suite on pull requests loads up metastation with a working holodeck area. We could alternatively fix this by having CI on local load metastation, just like github actions.

Reply:
I don't think that'd be a good fix. How come does it runtime for us and not for /tg/?

Reply:
Like, things like these should *absolutely not* silently runtime.

Reply:
We can add a return with an admin log printout about the holodeck not being complete on the map? If either linked's are null as it carries on?